addons/l10n_in/views/report_invoice.xml

```diff
@@ -30,6 +22,25 @@
             </td>
         </xpath>
 
+        <xpath expr="//div[@id='qrcode_info']" position="replace">
+            <t t-if="o.company_id.account_fiscal_country_id.code == 'IN'">
+                <div style="max-height:120px;">
+                    <strong>PAYMENT QR CODE</strong><br/>
+                    <div t-if="o.company_id.l10n_in_upi_id" class="mb-0">
+                        <p class="mb-0 mt-2">UPI ID: <br/>
+                        <p class="mb-0" t-field="o.company_id.l10n_in_upi_id"></p>
+                        </p>
+                    </div>
+                    <div class="d-flex mt--2" t-attf-style="#{'-webkit-transform:translateX(-0.5rem);' if report_type != 'html' else '-webkit-transform:translate(-0.5rem,-0.8rem);'}">
+                        <img t-att-src="'/l10n_in/static/src/img/PhonePe-Logo.svg'" style="width:4.5rem;"></img>
```

Comment:
Please use self closing tag 
(if you need doc: http://xahlee.info/js/html5_non-closing_tag.html)